### PR TITLE
Recorded Future Malware Hunt

### PR DESCRIPTION
**Pull request by RecordedFutureOskbo**
Required items, please complete
   
   Change(s):
   - Added (Recorded Future Malware Threat Map) Workbook
   - Added (ThreatMapMalware-Importer) Playbook.
   - Added (MalwareThreatHunt-IndicatorImport) Playbook.
   - Fix defaults on RecordedFuture-ActorThreatHunt-IndicatorImport Playbook
   - Fixed description on RecordedFutureThreatHuntingDomainAllActors Analytic Rules
   - Fixed description on RecordedFutureThreatHuntingHashAllActors Analytic Rules
   - Added Malware endpoints to RecordedFuture-CustomConnector Playbook
   - Fixed defaults on Playbook-Alert-Importer Playbook
   - Updated API connection names for all Playbooks to ease API connection configuration. Changed connectorId for Hunting Analytic Rules
   - Updated documentation

   Reason for Change(s):
   - New version 3.2

   Version Updated:
   - yes

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

### CODE REVIEW

**Comment by v-prasadboke** [comment]
Hello @RecordedFutureOskbo, Thanks for raising this PR. This PR will be investigated and will update you about the same before 30 January, 2024.

**Comment by v-prasadboke** [comment]
Hi @RecordedFutureOskbo, Can you provide me write access to the repo. I'm unable to pull the latest changes to my local.

**Comment by RecordedFutureOskbo** [comment]
> Hi @RecordedFutureOskbo, Can you provide me write access to the repo. I'm unable to pull the latest changes to my local.

You have write access to the fork since previous PRs?

**Comment by v-prasadboke** [comment]
Ok @RecordedFutureOskbo, I'll check once. 
Was facing issue while issue while shifting branches on github desktop

**Comment by RecordedFutureOskbo** [comment]
As always, we are open to teams calls or anything that make the process smoother.

**Comment by v-prasadboke** [comment]
Thank you for the help @RecordedFutureOskbo, I'll investigate the solution and if I stuck I'll definitely ask for a call. 
I'll update you before 06 February, 2024.

File: Workbooks/Images/Preview/RecordedFutureMalwareThreatHuntingBlack.png
Comment (v-prasadboke) [inline Workbooks/Images/Preview/RecordedFutureMalwareThreatHuntingBlack.png:1]: Hello @RecordedFutureOskbo, Threat Malware Category and recorded future watchlist are unset. And even i dont have the option to set those. is this due to lack of sample data? 
Can you share working image with setting this 2 options with something

**Comment by v-prasadboke** [comment]
**Recorded Future Malware Threat Map** and **Recorded Future - Threat Actor Hunting** shares common UI and query. 
Can you please check.

**Comment by RecordedFutureOskbo** [comment]
> **Recorded Future Malware Threat Map** and **Recorded Future - Threat Actor Hunting** shares common UI and query. Can you please check.

It's similar but pulls data from different tables **RecordedFutureThreatMap_CL** and the other
**RecordedFutureThreatMapMalware_CL**. We might merge them in to one, but I wanted to keep them separate in the first release.

File: Workbooks/Images/Preview/RecordedFutureMalwareThreatHuntingBlack.png
Comment (v-prasadboke) [reply Workbooks/Images/Preview/RecordedFutureMalwareThreatHuntingBlack.png:1]: > Hello @RecordedFutureOskbo, Threat Malware Category and recorded future watchlist are unset. And even i dont have the option to set those. is this due to lack of sample data? Can you share working image with setting this 2 options with something

anything on this

Comment (RecordedFutureOskbo) [reply Workbooks/Images/Preview/RecordedFutureMalwareThreatHuntingBlack.png:1]: Yes its lack of sample data, here is an image of filtering
image

Comment (v-prasadboke) [reply Workbooks/Images/Preview/RecordedFutureMalwareThreatHuntingBlack.png:1]: Thank you @RecordedFutureOskbo, That would help a lot

**Review by v-prasadboke — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**